nat-lab/bin/multicast.py

```diff
@@ -56,7 +67,10 @@ def mdns_client(timeout: int):
 def mdns_server(timeout: int):
     s = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
     s.settimeout(timeout)
-    s.setsockopt(socket.SOL_SOCKET, socket.SO_REUSEADDR, 1)
+    if sys.platform == "win32":
+        s.setsockopt(socket.SOL_SOCKET, socket.SO_REUSEADDR, 1)
+    else:
+        s.setsockopt(socket.SOL_SOCKET, socket.SO_REUSEPORT, 1)
```

Comment:
I can see the tests are passing so must not be an issue but will this not change the behavior on linux since it's now using `SO_REUSEPORT` instead of  `SO_REUSEADDR`? Or could it be that it doesn't matter for linux and it's just important that mac and windows get the right socket options? If so, I think the different cases should be standardized to make all them look like this one `if windows: else:`

Reply:
Uhh sorry, good catch, meant to do this the same as above